routes/api/report/conversion.js

```diff
@@ -0,0 +1,267 @@
+const Joi = require('joi');
+const async = require('async');
+
+exports.reportcsv = {
+  path: '/api/report/conversion.csv',
+  method: 'get',
+  config: {
+    validate: {
+      query: {
+        name: Joi.string().required(),
+        period: Joi.string().default('d'),
+        last: Joi.string(),
+      }
+    }
+  },
+  handler: {
+    autoInject: {
+      report(server, request, done) {
+        server.req.get('/api/report/conversion', { query: request.query }, done);
+      },
+      setHeaders: (request, done) => done(null, { 'content-type': 'application/csv' }),
```

Comment:
can we do the new style of autoInject, where we do `send(reply, ..., done)` and then set the headers there?